### DISCUSSION TITLE
Fails to start on download latest version

### DISCUSSION THREAD

**Discussion by jseifeddine** [netbox-community/netbox-docker#1318] [Getting Help]
### Current Behavior

netbox-1       | [ Use DB_WAIT_DEBUG=1 in netbox.env to print full traceback for errors here ]
netbox-1       | ⏳ Waiting on DB... (6s / 30s)
netbox-1       | django.db.utils.OperationalError: [Errno -2] Name or service not known
netbox-1       | [ Use DB_WAIT_DEBUG=1 in netbox.env to print full traceback for errors here ]
netbox-1       | ⏳ Waiting on DB... (9s / 30s)
netbox-1       | django.db.utils.OperationalError: [Errno -2] Name or service not known
netbox-1       | [ Use DB_WAIT_DEBUG=1 in netbox.env to print full traceback for errors here ]
netbox-1       | ⏳ Waiting on DB... (12s / 30s)
netbox-1       | django.db.utils.OperationalError: [Errno -2] Name or service not known
netbox-1       | [ Use DB_WAIT_DEBUG=1 in netbox.env to print full traceback for errors here ]
netbox-1       | ⏳ Waiting on DB... (15s / 30s)
netbox-1       | django.db.utils.OperationalError: [Errno -2] Name or service not known
netbox-1       | [ Use DB_WAIT_DEBUG=1 in netbox.env to print full traceback for errors here ]
netbox-1       | ⏳ Waiting on DB... (18s / 30s)
netbox-1       | django.db.utils.OperationalError: [Errno -2] Name or service not known
netbox-1       | [ Use DB_WAIT_DEBUG=1 in netbox.env to print full traceback for errors here ]
netbox-1       | ⏳ Waiting on DB... (21s / 30s)
netbox-1       | django.db.utils.OperationalError: [Errno -2] Name or service not known
netbox-1       | [ Use DB_WAIT_DEBUG=1 in netbox.env to print full traceback for errors here ]
netbox-1       | ⏳ Waiting on DB... (24s / 30s)
netbox-1       | django.db.utils.OperationalError: [Errno -2] Name or service not known
netbox-1       | [ Use DB_WAIT_DEBUG=1 in netbox.env to print full traceback for errors here ]
netbox-1       | ⏳ Waiting on DB... (27s / 30s)
netbox-1       | ❌ Waited 30s or more for the DB to become ready.

### Expected Behavior

Working 

### Docker Compose Version

Docker version 27.2.1, build 9e34c9b

### Docker Version

```text
Client: Docker Engine - Community
 Version:           27.2.1
 API version:       1.47
 Go version:        go1.22.7
 Git commit:        9e34c9b
 Built:             Fri Sep  6 12:08:06 2024
 OS/Arch:           linux/amd64
 Context:           default

Server: Docker Engine - Community
 Engine:
  Version:          27.2.1
  API version:      1.47 (minimum version 1.24)
  Go version:       go1.22.7
  Git commit:       8b539b8
  Built:            Fri Sep  6 12:08:06 2024
  OS/Arch:          linux/amd64
  Experimental:     false
 containerd:
  Version:          1.7.21
  GitCommit:        472731909fa34bd7bc9c087e4c27943f9835f111
 runc:
  Version:          1.1.13
  GitCommit:        v1.1.13-0-g58aa920
 docker-init:
  Version:          0.19.0
  GitCommit:        de40ad0
```

### The git Revision

commit 5add31ee7a55da8872efaa95278037a0e34020d5 (HEAD -> release, tag: 3.0.1, origin/release, origin/HEAD)

### The git Status

```text
On branch release
Your branch is up to date with 'origin/release'.
```

### Startup Command

docker compose up -d

### NetBox Logs

```text
netbox-1       | [ Use DB_WAIT_DEBUG=1 in netbox.env to print full traceback for errors here ]
netbox-1       | ⏳ Waiting on DB... (6s / 30s)
netbox-1       | django.db.utils.OperationalError: [Errno -2] Name or service not known
netbox-1       | [ Use DB_WAIT_DEBUG=1 in netbox.env to print full traceback for errors here ]
netbox-1       | ⏳ Waiting on DB... (9s / 30s)
netbox-1       | django.db.utils.OperationalError: [Errno -2] Name or service not known
netbox-1       | [ Use DB_WAIT_DEBUG=1 in netbox.env to print full traceback for errors here ]
netbox-1       | ⏳ Waiting on DB... (12s / 30s)
netbox-1       | django.db.utils.OperationalError: [Errno -2] Name or service not known
netbox-1       | [ Use DB_WAIT_DEBUG=1 in netbox.env to print full traceback for errors here ]
netbox-1       | ⏳ Waiting on DB... (15s / 30s)
netbox-1       | django.db.utils.OperationalError: [Errno -2] Name or service not known
netbox-1       | [ Use DB_WAIT_DEBUG=1 in netbox.env to print full traceback for errors here ]
netbox-1       | ⏳ Waiting on DB... (18s / 30s)
netbox-1       | django.db.utils.OperationalError: [Errno -2] Name or service not known
netbox-1       | [ Use DB_WAIT_DEBUG=1 in netbox.env to print full traceback for errors here ]
netbox-1       | ⏳ Waiting on DB... (21s / 30s)
netbox-1       | django.db.utils.OperationalError: [Errno -2] Name or service not known
netbox-1       | [ Use DB_WAIT_DEBUG=1 in netbox.env to print full traceback for errors here ]
netbox-1       | ⏳ Waiting on DB... (24s / 30s)
netbox-1       | django.db.utils.OperationalError: [Errno -2] Name or service not known
netbox-1       | [ Use DB_WAIT_DEBUG=1 in netbox.env to print full traceback for errors here ]
netbox-1       | ⏳ Waiting on DB... (27s / 30s)
netbox-1       | ❌ Waited 30s or more for the DB to become ready.
```

### Content of docker-compose.override.yml

```yaml
cat docker-compose.override.yml 
services:
  netbox:
    ports:
      - 8000:8080
```

**Comment by tobiasge** [comment]
Have you set `DB_WAIT_DEBUG=1` to see what the full logs are? Is your database up and reachable?

**Reply by jseifeddine** [reply]
db is up.. its just postgres as so:

```yaml
  postgres:
      restart: always
      image: bitnami/postgresql:latest
      environment:
        - POSTGRESQL_USERNAME=netbox
        - POSTGRESQL_DATABASE=netbox
        - POSTGRESQL_PASSWORD=what
        - POSTGRESQL_POSTGRES_PASSWORD=w0t
      volumes:
        - postgresql:/bitnami/postgresql
 ```

i use the same `service` snippet for other stuff and postgres works fine, tested also working 

```bash
docker run -it --rm bitnami/postgresql:latest psql -h 172.20.2.11 -U netbox -d netbox
postgresql 19:42:11.13 INFO  ==> 
postgresql 19:42:11.14 INFO  ==> Welcome to the Bitnami postgresql container
postgresql 19:42:11.15 INFO  ==> Subscribe to project updates by watching https://github.com/bitnami/containers
postgresql 19:42:11.16 INFO  ==> Submit issues and feature requests at https://github.com/bitnami/containers/issues
postgresql 19:42:11.16 INFO  ==> Upgrade to Tanzu Application Catalog for production environments to access custom-configured and pre-packaged software components. Gain enhanced features, including Software Bill of Materials (SBOM), CVE scan result reports, and VEX documents. To learn more, visit https://bitnami.com/enterprise
postgresql 19:42:11.17 INFO  ==> 

Password for user netbox: 
psql (16.4)
Type "help" for help.

netbox=> 
```

**Reply by jseifeddine** [reply]
All good, you made me think, 

I used the postgres user instead of netbox - must be a permissions issue

plz close
